src/documents/management/commands/manage_superuser.py

```diff
@@ -1,42 +1,64 @@
 import logging
 import os
-
+import re
 from django.contrib.auth.models import User
 from django.core.management.base import BaseCommand, CommandError
-
+from django.forms import EmailField
+from django.core.exceptions import ValidationError
 
 logger = logging.getLogger("paperless.management.superuser")
 
 
+def isEmailAddressValid(email):
+    try:
+        EmailField().clean(email)
```

Comment:
This even works with root@localhost as localhost is already in the OOTB domain_allowlist.
```suggestion
        EmailValidator()(email)
```